src/test/java/com/google/sps/GetConfigPropertiesTest.java

```diff
@@ -0,0 +1,44 @@
+
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class GetConfigPropertiesTest {
+  @Test
+  public void GetApiKeyString() throws FileNotFoundException, IOException {
+    String actual = GetConfigProperties.getApiKey();
+    String expected = "mockApiKey";
+    Assert.assertEquals(expected, actual);
+  }
+
+  @Test
+  public void CheckIfPropertiesFileExist() {
+    String rootPath = Thread.currentThread().getContextClassLoader().getResource("").getPath();
+    Path path = Paths.get(rootPath + "app.properties");
+    Assert.assertTrue(Files.exists(path));
+  }
+}
```

Comment:
replying to your comment about tests here because I can't figure out how to do it it directly.
I think ideally you would also include tests where the resource folder is missing the app.properties file or where the file is missing the appropriate key. 
The way you've written the function where the path is basically hard coded makes this difficult, you would basically have to change the context class for each test, which I think is possible but probably not worth getting into right now. 
An alternative is to have two library functions, one that extracts a given key from a given file and one that has the hard coding, eg GetPropertyFromConfig(filepath, propertyName), and GetApiKeyString(). that way you can test the failure cases on the general library function by passing in a non-existent file path or a different file path that is missing the key, and you can test the success case with the function that has the hard coded path.
I wouldn't spend too much time on this because I do want you to get on to the api integration, but if you can do this in an hour or so I think it would be a worthwhile thing to learn how to structure you code so it's easier to test.